sklearn/impute/_base.py

```diff
@@ -597,8 +597,10 @@ def fit(self, X, y=None):
 
         Returns
         -------
-        self : object
-            Returns self.
+        imputer_mask : {ndarray or sparse matrix},
```

Comment:
```py
        imputer_mask : {ndarray or sparse matrix}, shape (n_samples, \
        n_features)
            The imputer mask of the original data.
```

Reply:
Is something wrong?

Reply:
The spacing. This is only for consistency. This docstring is not going to be rendered since `_fit` is private.

Notice the use of `\` and how `n_features)` is aligned and how many white spaces there are before `The imputer...`.

Reply:
Done.

Thanks.